app/javascript/dashboard/assets/scss/widgets/_reports.scss

```diff
@@ -0,0 +1,30 @@
+.date-picker {
+  margin-left: var(--space-smaller);
+}
+
+.margin-left-small {
+  margin-left: var(--space-smaller);
+}
+
+.reports-option__rounded--item {
+  border-radius: 100%;
+  height: var(--space-two);
+  width: var(--space-two);
+}
+
+.reports-option__item {
+  flex-shrink: 0;
+  margin-right: var(--space-small);
+}
+
+.reports-option__label--swatch {
+  border: 1px solid #fff;
```

Comment:
Use CSS variable

Reply:
Fixed in https://github.com/chatwoot/chatwoot/pull/3084/commits/cb237fd3bd035954288c48f074e3308da2cb9815